src/ValueFormatters/QuantityHtmlFormatter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ValueFormatters;
+
+/**
+ * HTML formatter for quantity values.
+ *
+ * @since 0.5.1
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class QuantityHtmlFormatter extends QuantityFormatter {
+
+	/**
+	 * @param string $number
+	 * @param string $unit
+	 *
+	 * @return string HTML
+	 */
+	protected function applyUnit( $number, $unit ) {
+		$html = htmlspecialchars( $number );
+
+		if ( $this->vocabularyUriFormatter !== null ) {
+			$unit = $this->vocabularyUriFormatter->format( $unit );
+
+			if ( $unit === null ) {
+				return $html;
+			}
+		}
+
+		return $html . ' <span class="unit">' . htmlspecialchars( $unit ) . '</span>';
```

Comment:
* Data attribute: Could do, but why? Not relevant for the things we want to do in the sprint. YAGNI.
* A "configurable" CSS class name? Who, no. The fact that it's "generic" is intended. Code that applies styles on this without a proper selector is wrong anyway. But I agree that we should change this to "wb-unit".